# HG changeset patch
# Date 1359974819 -3600
# Parent 2c384745f5e07e97e653cfa1faea15213621bf16
# User Philipp Wagner <mail@philipp-wagner.com>
Disable option checking and bzip2 support for the in-tree freetype.

Option checking prevents configure from being called with unknown options.
Since we pass all Mozilla configure options to Freetype, this causes a
configure error even though everything works as expected.

bzip2 support gives a linker error when linking libxul.so since bz2
is not linked in. The real solution would probably be to include bz2
when linking.

diff --git a/configure.in b/configure.in
--- a/configure.in
+++ b/configure.in
@@ -9251,17 +9251,17 @@ fi
 # Run freetype configure script
 
 if test "$MOZ_TREE_FREETYPE"; then
    export CFLAGS="$CFLAGS $MOZ_DEBUG_FLAGS -std=c99"
    export CPPFLAGS="$CPPFLAGS $MOZ_DEBUG_FLAGS"
    export CXXFLAGS="$CXXFLAGS $MOZ_DEBUG_FLAGS"
    export LDFLAGS="$LDFLAGS $MOZ_DEBUG_LDFLAGS"
    export CONFIG_FILES="unix-cc.mk:unix-cc.in unix-def.mk:unix-def.in freetype-config freetype2.pc:freetype2.in"
-   ac_configure_args="$ac_configure_args --host=$target --disable-shared --with-pic=yes"
+   ac_configure_args="--disable-option-checking $ac_configure_args --host=$target --disable-shared --with-pic=yes --without-bzip2"
    AC_OUTPUT_SUBDIRS(modules/freetype2)
 fi
 
 if test -z "$direct_nspr_config"; then
     dnl ========================================================
     dnl = Setup a nice relatively clean build environment for
     dnl = sub-configures.
     dnl ========================================================
